Renderer.js

```diff
@@ -0,0 +1,43 @@
+class Renderer {
+  appendUserData(data) {
+    const source = $('#user-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.user-container').empty().append(newHTML);
+  }
+
+  appendKenyaData(data) {
+    const source = $('#quote-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.quote-container').empty().append(newHTML);
+  }
+
+  appendPokemonData(data) {
+    const source = $('#pokemon-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.pokemon-container').empty().append(newHTML);
+  }
+
+  appendBaconData(data) {
+    const source = $('#meat-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.meat-container').empty().append(newHTML);
+  }
+
+  appendFriends(data) {
+    const source = $('#friends-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.friends-container').empty().append(newHTML);
+  }
+
+  appendDropdown(data) {
+    const source = $('#dropdown-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(data);
+    $('.saved-users').empty().append(newHTML);
+  }
+}
```

Comment:
note that all the functions here are basically the same - the only difference between each of them is the first line -->> you could make one function which will get a second parameter, like templateName, beside the data parameter, and use that parameter for the first line of code (the - const source = $(`#${templateName}`).html() )